orca.lua

```diff
@@ -706,7 +715,33 @@ local kb = {
   s = {[42] = true, [54] = true},
   c = {[29] = true, [125] = true, [127] = true, [97] = true}
 }
-function keyboard.event(typ, code, val)
+
+
+local is_normal_k_pressed = false
+keyboard.char = function(a)
+  local keyinput = a
+  if is_normal_k_pressed then
+    if not ctrl then
+      if orca.cell[y_index][x_index] == "/" then
+        orca.sc_ops.count = util.clamp(orca.sc_ops.count - 1, 1, 6)
+      end
+      cell_input = keyinput
+      orca.cell[y_index][x_index] = keyinput
+    elseif ctrl then
+      if a == 'x' then
+        orca:copy_area(x_index, y_index, true)
+      elseif a == 'c' then
+        orca:copy_area(x_index, y_index)
+      elseif a == 'v' then
+        orca:paste_area(x_index, y_index)
+      end
+    end
+  end
+end
+
+keyboard.code = function(c, val)
+  local code = find_in_table(c, keyboard.codes)
```

Comment:
hackish but least intrusive way to migrate.

`keyboard` now provides the way more legible `keyboard.state.<KEY>`.